Makefile

```diff
@@ -40,11 +40,12 @@ all: init
 doc: init
 	$(MAKE) -C $(BUILD_DIR) doc
 
-install: all
+install: | uninstall all
```

Comment:
I'd just call uninstall as first command in this target:
install: all
   $(MAKE) uninstall
  ....